spyder/workers/updates.py

```diff
@@ -236,7 +237,7 @@ def _download_installer(self):
         # platforms will be x86_64.
         mach = "arm64" if "ARM64" in platform.version() else "x86_64"
 
-        if parse(self.latest_release_version) > parse("5"):
+        if parse(self.latest_release_version) >= parse("6"):
```

Comment:
```suggestion
        if parse(self.latest_release_version) >= parse("6.0.0"):
```

Let's be more explicit here, just in case.